src/main/java/net/minestom/server/event/inventory/InventoryCloseEvent.java

```diff
@@ -34,10 +34,9 @@ public Player getPlayer() {
     /**
      * Gets the new inventory to open.
      *
-     * @return the new inventory to open, null if there isn't any
```

Comment:
why is this comment removed? there is still a null case here

Reply:
It's not null when being passed to the constructor, the only null case is if somebody explicitly calls `setNewInventory(null)`. I am unclear on what this does or why you can even do that.

Reply:
`setNewInventory(null)` is for indicating that you don't want a new inventory to be opened when one is closed

Reply:
`null if no inventory will be opened after closing this one` or something seems more clear to me.